JCoreRN.podspec

```
@@ -3,7 +3,7 @@ pjson = JSON.parse(File.read('package.json'))
 
 Pod::Spec.new do |s|
 
-  s.name            = "JCore"
+  s.name            = "JCoreRN"
```

RuboCop (1.81.1)

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)